src/OrchardCore.Modules/OrchardCore.Lists/Views/ListPartNavigationAdmin.cshtml

```diff
@@ -4,7 +4,7 @@
 @inject OrchardCore.ContentManagement.IContentManager ContentManager
 @inject Microsoft.AspNetCore.Authorization.IAuthorizationService AuthorizationService
 
-<div class="card bg-primary text-right mb-3" style="z-index: 1001">
+<div class="card bg-primary text-right mb-3" style="z-index: 1000">
```

Comment:
This should use a class instead of inline styles. Should be also using Bootstrap z-index if we can.